packages/frontend/src/components/common/Table/Table.styles.ts

```diff
@@ -39,69 +40,117 @@ export const TableContainer = styled.div<TableContainerProps>`
             `}
 `;
 
-export const Table = styled(HTMLTable)<{ $showFooter: boolean }>`
+export const Table = styled.table<{ $showFooter: boolean }>`
+    border-spacing: 0;
+    font-size: 14px;
     background-color: white;
     width: 100%;
     border-left: 1px solid #dcdcdd;
     border-right: 1px solid #dcdcdd;
 
+    th,
+    td {
+        padding-left: 11px;
+        padding-right: 11px;
+        padding-bottom: 6px;
+        padding-top: 6px;
+        text-align: left;
+        vertical-align: top;
+    }
+
+    th {
+        color: #1c2127;
+        font-weight: 600;
+    }
+    td {
+        color: #1c2127;
+    }
+    tbody tr:first-child th,
+    tbody tr:first-child td,
+    tfoot tr:first-child th,
+    tfoot tr:first-child td {
+        box-shadow: inset 0 1px 0 0 rgba(17, 20, 24, 0.15);
+    }
+
+    tbody tr td,
+    tfoot tr td {
+        box-shadow: inset 0 1px 0 0 rgba(17, 20, 24, 0.15);
+    }
+
+    tbody tr td:not(:first-child),
+    tfoot tr td:not(:first-child) {
+        box-shadow: inset 1px 1px 0 0 rgba(17, 20, 24, 0.15);
+    }
+
+    tbody tr:nth-child(odd) td {
+        background: rgba(143, 153, 168, 0.15);
+    }
+
+    th:not(:first-child) {
+        box-shadow: inset 1px 0 0 0 rgba(17, 20, 24, 0.15);
+    }
+
+    tbody tr:hover td {
+        background-color: rgba(143, 153, 168, 0.3);
+        cursor: pointer;
+    }
+    tbody tr:active td {
+        background-color: rgba(143, 153, 168, 0.35);
+    }
+
+    /* FIXME: everything above this line is copied from blueprint's table css */
```

Comment:
Do you know what this fixme will require? Like, will we move to a new table or just keep this css forever?

Reply:
@magnew 
I think we should migrate to `LightTable` (component that Pivot tables use)
should be a fun lightsmash friday task